modal/_asgi.py

```diff
@@ -167,3 +170,194 @@ def webhook_asgi_app(fn: Callable, method: str):
     )
     app.add_api_route("/", fn, methods=[method])
     return app
+
+
+def get_ip_address(ifname: bytes):
+    """Get the IP address associated with a network interface in Linux."""
```

Comment:
why do you need the ip address?

Reply:
It's documented at the site of usage: https://github.com/modal-labs/modal-client/pull/1513/files#diff-af590c8ac1ecd84881361705179ff347e2f4bfbe34549d810b5a6eeb83789a06R895

By grabbing the external IP of the container on the network gateway we can have the HTTP traffic routed `worker → container web server` instead of `worker → container modal client → container web server`.